gnomad_qc/v5/annotations/compute_coverage.py

```diff
@@ -0,0 +1,1054 @@
+    telomeres_and_centromeres,
+    vep_context,
+)
+from gnomad.utils.annotations import (
+    annotate_downsamplings,
+    build_freq_stratification_list,
+    generate_freq_group_membership_array,
+    merge_array_expressions,
+    merge_histograms,
+    qual_hist_expr,
+)
+from gnomad.utils.sparse_mt import (
+    compute_stats_per_ref_site,
+    get_allele_number_agg_func,
+    get_coverage_agg_func,
+)
+from hail.utils.misc import new_temp_file
+
+from gnomad_qc.resource_utils import check_resource_existence
+from gnomad_qc.v4.resources.meta import meta as v4_meta
+from gnomad_qc.v5.annotations.annotation_utils import annotate_adj
+from gnomad_qc.v5.resources.annotations import (
+    coverage_and_an_path,
+    get_aou_downsampling,
+    group_membership,
+    qual_hists,
+)
+from gnomad_qc.v5.resources.basics import (
+    get_aou_vds,
+    get_gnomad_v5_genomes_vds,
+    get_logging_path,
+    qc_temp_prefix,
+)
+from gnomad_qc.v5.resources.constants import WORKSPACE_BUCKET
+
+# from gnomad_qc.v5.resources.meta import meta
+from gnomad_qc.v5.resources.release import (
+    release_all_sites_an_tsv_path,
+    release_coverage_path,
+    release_coverage_tsv_path,
+)
+
+logging.basicConfig(format="%(levelname)s (%(name)s %(lineno)s): %(message)s")
+logger = logging.getLogger("aou_coverage_and_an")
+logger.setLevel(logging.INFO)
+
+
+def get_downsampling_ht(ht: hl.Table) -> hl.Table:
+    """
+    Get Table with downsampling groups for all samples.
+
+    v5 downsampling is only applied to the AoU dataset.
+    Desired groups:
+    - 10,000
+    - 100,000
+    - Genetic ancestry group sizes for AFR, AMR, NFE
+    Note that the only desired genetic ancestry group sizes are AFR, AMR, and NFE,
+    but code will also generate downsamplings for all other groups.
+
+    :param ht: Input Table.
+    :return: Table with downsampling groups.
+    """
+    logger.info(
+        "Determining downsampling groups for AoU...",
+    )
+    downsamplings = DOWNSAMPLINGS["v5"]
+    ht = annotate_downsamplings(
+        ht, downsamplings, ht.genetic_ancestry_inference.gen_anc
+    )
+    return ht
+
+
+def get_group_membership_ht(
+    meta_ht: hl.Table, project: str, ds_ht: Optional[hl.Table] = None
+) -> hl.Table:
+    """
+    Get genomes group membership HT for all sites allele number stratification.
+
+    :param meta_ht: Meta HT.
+    :param project: Project name.
+    :param ds_ht: Optional downsampling HT. Only used for AoU.
+    :return: Group membership HT.
+    """
+    if project == "aou":
+        ht = generate_freq_group_membership_array(
+            meta_ht,
+            build_freq_stratification_list(
+                sex_expr=meta_ht.sex_imputation.sex_karyotype,
+                gen_anc_expr=meta_ht.genetic_ancestry_inference.gen_anc,
+                downsampling_expr=ds_ht[meta_ht.key].downsampling,
+            ),
+            downsamplings=hl.eval(ds_ht.downsamplings),
+            ds_gen_anc_counts=hl.eval(ds_ht.ds_gen_anc_counts),
```

Comment:
i didn't overwrite the downsampling ht, but the gen_anc_counts in gs://fc-secure-b25d1307-7763-48b8-8045-fcae9caadfa1/tmp/gnomad.genomes.v5.0.qc_data/sample_qc/genomes/aou/gnomad.genomes.v5.0.group_membership.ht don't seem to match the gen_anc counts obtained from the metadata after filtering to project and release

Reply:
I think it is because I ran using an outdated meta, since I ran this when `sex_karyotype` was still nested within `sex_imputation`. the counts in the meta HT at that time were 
<img width="835" height="320" alt="image" src="https://github.com/user-attachments/assets/d4e4c171-d928-4f22-a1bf-464b03a943a9" />
which seems to match the downsampling ht counts
<img width="400" height="182" alt="image" src="https://github.com/user-attachments/assets/34c014b2-e261-434e-8759-557671a54dd8" />

but I'll rerun just to make sure the code works after pushing changes and check again